gentoo/gentoo
# Various gnucash fixes

- **Don't build gtest from source**
With gnucash-3.6, the build now looks for gtest headers and libraries and uses them if found.
Closes: https://bugs.gentoo.org/691276

- **Fix testcase failures**
test-qof and test-gnc-numeric both depend on the existence of en_US, en_GB, and fr_FR locales and fail if not installed. Check the output of `locale -a` and disable the tests if not found.

- **Bump gnucash-docs-3.7**

- **Bump gnucash-3.7**

- **Fix check_symbol_exists error**
Adds `include(CheckSymbolExists)` to avoid the error `Unknown CMake command check_symbol_exists`
Closes: https://bugs.gentoo.org/690270

- **Remove unused cmake variable**
Closes: https://bugs.gentoo.org/689744

- **Disable auto-compiling /usr/bin/guild**
Stop the build from unnecessarily attempting to compile /usr/bin/guild. Prevents messages like:
```
;;; note: auto-compilation is enabled, set GUILE_AUTO_COMPILE=0
;;;       or pass the --no-auto-compile argument to disable.
;;; compiling /usr/bin/guild
;;; WARNING: compilation of /usr/bin/guild failed:
;;; failed to create path for auto-compiled file "/usr/bin/guild"
```

- **Fix testcases that write to /tmp**
Have testcases write to "${T}" instead of /tmp

- **Fix sys-libs/gwenhywfar dependency**
Building with aqbanking and gui flags relies on >=sys-libs/gwenhywfar-4.20.2[gtk]. Without it, when building with gui and aqbanking USE flags, gnucash tries to build bundled libgwengui-gtk3.so.

- **Rename chipcard USE flag to smartcard**
Closes: https://bugs.gentoo.org/682120

- **sys-libs/gwenhywfar: block <app-office/gnucash-3.7[aqbanking]**
<app-office/gnucash-3.7[aqbanking] unfortunately provided its own libgwengui-gtk3.so which conflicts with that provided by sys-libs/gwenhywfar[gtk].
Closes: https://bugs.gentoo.org/695386

- **Replace gnome2_icon_cache_update**

Closes: #12792
Package-Manager: Portage-2.3.72, Repoman-2.3.17
Signed-off-by: Peter Levine <plevine457@gmail.com>
